web/src/upload.js

```diff
@@ -411,14 +411,25 @@ export function setup_upload(config) {
 
 export function initialize() {
     // Allow the main panel to receive drag/drop events.
-    $(".app-main").on("dragover", (event) => event.preventDefault());
+    $(".app-main").on("dragover", (event) => {
+        event.preventDefault();
+        $(".blur-overlay").css("display", "block");
```

Comment:
Instead of adding CSS property in Javascript, you can either use `show`/`hide` methods of jquery or you can add and remove some class using `addClass`/`removeClass` functions here and use that class to define CSS properties. We already use "hide" class for such use cases in many places.

Reply:
thanks for the review, I will make changes according to what you have mentioned